ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -103,53 +149,28 @@ open class CommandBar: UIView {
         scrollView.translatesAutoresizingMaskIntoConstraints = false
         scrollView.contentInset = UIEdgeInsets(
```

Comment:
in the RTL scenarios shouldn't the spacing be flipped?

Reply:
I believe the transform used on the `scrollView` in `configureHierarchy()` takes care of flipping the content insets. I tested this locally using a larger value for `CommandBar.fixedButtonSpacing` I was able to see the shift of the content inset from one side to the other as I changed from an LTR language to an RTL language.